lib/cmath.rb

```diff
@@ -1,16 +1,49 @@
 ##
+# = Trigonometric and transcendental functions for complex numbers.
+#
 # CMath is a library that provides trigonometric and transcendental
-# functions for complex numbers.
+# functions for complex numbers. The functions in this module accept
+# integers, floating-point numbers or complex numbers as arguments. 
+#
+# Note that the selection of functions is similar, but not identical,
+# to that in module math. The reason for having two modules is that
+# some users aren’t interested in complex numbers, and perhaps don’t
+# even know what they are. They would rather have Math.sqrt(-1) raise
+# an exception than return a complex number.
+#
+# == Brief overview of complex numbers
```

Comment:
@davydovanton Thank you for updating! How about having this section at `Complex` and putting link to `Complex` here?
sorry I just wanna avoid duplicate contents.